src/types/solidity-types.ts

```diff
@@ -38,6 +38,8 @@ export function getVisitableKeys(nodeType: string): string[] {
       return ['components'];
     case 'FunctionCall':
       return ['expression', 'arguments'];
+    case 'InternalFunctionCall' :
+      return ['name', 'internalFunctionInteractsWithSecret','parameters'];
```

Comment:
`internalFunctionInteractsWithSecret` isn't a visitable key btw - these keys are only things that the visitor should look at to find the next node in the tree